api/handler/v1/runtime.go

```diff
@@ -12,12 +12,12 @@ import (
 	"google.golang.org/protobuf/types/known/structpb"
 
 	"github.com/odpf/optimus/core/tree"
+	lg "github.com/odpf/salt/log"
```

Comment:
Why use an alias?

Reply:
As direct implementation through salt is by `log` keyword like `log.Newlogrus()` , which may confuse us with `simple logging package` .If it is not an meaningful issue , will inject it without alias

Reply:
Removing alias